README.md

```diff
@@ -2,12 +2,11 @@
 
 [![Build status](https://badge.buildkite.com/921dc61e2d3a350ec40efb291914360c0bfa9b6196fa357420.svg?branch=master)](https://buildkite.com/bazel/bazel-skylib)
 
-Skylib is a standard library that provides functions useful for manipulating
-collections, file paths, and other features that are useful when writing custom
-build rules in Bazel.
+Skylib is a library of Starlark functions for manipulating collections, file paths,
+and various other data types in the domain of Bazel build rules.
 
-> This library is currently under early development. Be aware that the APIs
-> in these modules may change during this time.
+This library is currently under early development. Be aware that the APIs
```

Comment:
Perhaps it is time to change this statement.

It seems to me like the motivation behind this PR is that things added here essentially live forever, so we have to be careful about what is added. That is at odds with fast changing APIs.

Reply:
Indeed. Removed.